### ISSUE TITLE
Error when keep_tree=true in dpw.jl

### ISSUE THREAD

**Issue by rcnlee** [JuliaPOMDP/MCTS.jl#32]
I get this error when setting ``keep_tree=true``.  You can reproduce this by adding ``keep_tree=true`` in dpw_test.jl and then running ``Pkg.test("MCTS")``.  

It looks like those ``p.tree`` uses in dpw.jl at lines 29-32 should be ``tree`` instead.  I'll make the correction and submit a pull request.

```julia
Testing DPW solver.
ERROR: LoadError: LoadError: type Nullable has no field s_lookup
Stacktrace:
 [1] action(::MCTS.DPWPlanner{POMDPModels.GridWorld,POMDPModels.GridWorldState,Symbol,MCTS.SolvedRolloutEstimator{MersenneTwister},MCTS.RandomActionGenerator{MersenneTwister},MersenneTwister}, ::POMDPModels.GridWorldState) at /Users/rlee18/.julia/v0.6/MCTS/src/dpw.jl:29
 [2] include_from_node1(::String) at ./loading.jl:576
 [3] include(::String) at ./sysimg.jl:14
 [4] include_from_node1(::String) at ./loading.jl:576
 [5] include(::String) at ./sysimg.jl:14
 [6] process_options(::Base.JLOptions) at ./client.jl:305
 [7] _start() at ./client.jl:371
while loading /Users/rlee18/.julia/v0.6/MCTS/test/dpw_test.jl, in expression starting on line 22
while loading /Users/rlee18/.julia/v0.6/MCTS/test/runtests.jl, in expression starting on line 43
...
ERROR: MCTS had test errors
```

**Comment by rcnlee** [comment]
``sol.keep_tree || sol.check_repeat_state`` will always be true in the following code in dpw.jl.  My PR simplifies this to ``true``.  Is that the intended behavior?

```julia
if p.solver.keep_tree
        tree = get(p.tree, DPWTree{S,A}(p.solver.n_iterations))
        if haskey(p.tree.s_lookup, s)
            snode = p.tree.s_lookup[s]
        else
            snode = insert_state_node!(get(p.tree), s, sol.keep_tree || sol.check_repeat_state)
        end
    else
```

**Comment by zsunberg** [comment]
this was fixed by #33 Thanks @rcnlee !
